integration/110_shutdown_test.sh

```diff
@@ -5,12 +5,15 @@
 start_suite "Check scope exits cleanly within 10 seconds"
 
 scope_on $HOST1 launch
+sleep 10
```

Comment:
I thought 3 would be the conservative option; the non-conservative one being 1 :)

Anyway, LGTM